src/main/java/uk/gov/laa/ccms/caab/bean/ClientDetails.java

```diff
@@ -54,5 +56,24 @@ public class ClientDetails {
   private String disability;
   private String specialConsiderations;
 
+  /**
+   * Retrieves the formatted date of birth based on the day, month, and year values.
+   *
+   * @return The formatted date of birth (yyyy-MM-dd), or null if the date components are not valid
+   *         integers.
+   */
+  public String getDateOfBirth() {
+    try {
+      int year = Integer.parseInt(dobYear);
+      int month = Integer.parseInt(dobMonth);
+      int day = Integer.parseInt(dobDay);
+
+      return String.format("%d-%02d-%02d", year, month, day);
+    } catch (NumberFormatException e) {
+      // Handle the exception if any of the dobYear, dobMonth, or dobDay is not a valid integer
+      return null;
```

Comment:
No issues with returning a null rather than throwing an exception?

Reply:
This should always be populated as it comes from the client details search, which uses a validatoor to check the fields cannot be nullable. 

I will make it throw an error